### PR TITLE
Added .desktop file

### LINKED ISSUE #2: add .desktop file to open it from menu
it would be good to add a desktop file and icon.
so it will be accessible from menu.

### PR DESCRIPTION
**Pull request by gg-rewrite**
This closes #2

### CODE REVIEW

**Comment by gg-rewrite** [comment]
The last time was positively weird, here's the new one.

File: meson.build [hunk 1]
```diff
@@ -23,3 +23,8 @@ add_project_arguments(['-Wno-unused-parameter'], language: 'cpp')
 
 subdir('proto')
 subdir('src')
+
+install_data(
+  'wf-osk.desktop',
+  install_dir: '/usr/share/applications'
```
Comment (ammen99) [inline meson.build]: Shouldn't this be something like `join_paths(get_option('prefix'), 'share/applications')`?

Comment (ammen99) [reply meson.build]: Because the user may wish to install to other location than /usr

Comment (gg-rewrite) [reply meson.build]: And done

**Review by ammen99 — APPROVED** [review]
thanks :)

**Pull request merged**